google-cloud-storage/pom.xml

```diff
@@ -88,6 +88,11 @@
       <groupId>org.threeten</groupId>
       <artifactId>threetenbp</artifactId>
     </dependency>
+    <dependency>
```

Comment:
isn't this included in java 6 and later?

Reply:
it works with jdk7,8. This dependency is required for java9+